robot_skills/src/robot_skills/mockbot.py

```diff
@@ -193,6 +193,7 @@ def __init__(self, robot_name, tf_buffer, *args, **kwargs):
         self.local_planner = AlteredMagicMock()
         self.local_planner.getStatus = mock.MagicMock(return_value="arrived")  # always arrive for now
         self.global_planner.getPlan = mock.MagicMock(return_value=["dummy_plan"])  # always arrive for now
+        self.global_planner.path_length = 0.0 
```

Comment:
This is needed because navigation will fail otherwise. As it would try to compare a magicmock and a number

Reply:
Should this not also be a MagicMock?